### PR TITLE
Remove jenkins cookbook

### PR DESCRIPTION
**Pull request by anguss00**
https://github.com/aodn/backlog/issues/1776

### CODE REVIEW

File: cookbooks/imos_artifacts/libraries/fetcher_jenkins.rb [hunk 1]
```diff
@@ -1,198 +0,0 @@
-#
```
Comment (lwgordonimos) [inline cookbooks/imos_artifacts/libraries/fetcher_jenkins.rb:1]: I'd leave the imos_artifacts changes for another PR, and just get rid of the imos_jenkins cookbook in this one, because they are separate things really.

Comment (anguss00) [reply cookbooks/imos_artifacts/libraries/fetcher_jenkins.rb:1]: No worries, I'll back it out and make another PR

Comment (anguss00) [reply cookbooks/imos_artifacts/libraries/fetcher_jenkins.rb]: @lwgordonimos done, separate PR coming
